modules/internal/ChapelRange.chpl

```diff
@@ -16,21 +16,98 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/*
+  The ChapelRange module provides the implementation for the built in range
```

Comment:
Agreed, this sentence can be removed.

@vasslitvinov The `fixTitle` helper routine will take care of renaming "module::ChapelRange" to just "Range". It was added to do exactly as you're saying: abstract the actual implementation details and just refer to the type.